src/OpenApi/sample/Sample.csproj

```
@@ -4,6 +4,11 @@
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
+    <IsTrimmable>true</IsTrimmable>
```

```suggestion
    <IsAotCompatible>true</IsAotCompatible>
```

Although, this is a bit weird because this is an app, not a library.